dev/release/verify-release-candidate.sh

```diff
@@ -468,6 +468,22 @@ install_conda() {
   conda activate base
 }
 
+maybe_setup_maven() {
```

Comment:
Could you split `maybe_setup_maven` and `install_maven` like `maybe_setup_nodejs` and `install_nodejs`?

Could you also show `Ensuring ...` message like `maybe_setup_nodejs`?

Could you also check `USE_CONDA` like `maybe_setup_nodejs`?